wasi/2023/WASI-07-27.md

```diff
@@ -31,3 +31,223 @@ The meeting will be on a zoom.us video conference.
+
+Updating the text to have the same structure as the CG phases.
+
+#### Preview 2 Definition Plan
+
+**Dan Gohman**: WebAssembly/WASI#550
+
+This will become the document that defines what preview 2 is. First thing in this document is the Launch criteria AKA how do we know when this is done? Let’s talk about this instead in how a window is opened.
+
+Preview 2 will be considered launched once preview 2 has two worlds advanced to phase 3.
+
+We will live this window open until we wind down preview 2 for preview 3.
+
+We want to be able to show that we have at least two different worlds. This is important to show that WASI is about more than just POSIX. Are all the engines expected to implement all of the APIs? The vision for worlds is that not all engines will have to implement all of these worlds. Some engines may implement all of the worlds, but they would not be required to.
+
+Side note: the two proposals that are the furthest along are WASI CLI and WASI HTTP Proxy worlds. Those are the furthest along but it’s not specifically defined as being those two. The only requirement is that it’s at least two worlds.
+
+WASI Preview 2 Contents is a table that currently contains no elements. As proposals reach phase 3, then we will add them to the table.
+
+This will be as official as a markdown document in GitHub can be in that it’s not an official standard but a statement of stability.
+
+What does it mean to be part of this list? Reach phase 3, as I talked about in the previous agenda item. It must satisfy its own phase 4 acceptance criteria. Phase 4 in the core CG is effectively standardization. We are holding off on that for now, so we are defining a preview and creating a place to grow and evolve things.
+
+In order to define the contents of Preview 2, it must be voted for inclusion by the WASI Subgroup.
+
+Proposals may be added to WASI Preview 2 at any time until Preview 3.
+Any questions?
+
+**Conrad Watt**: It has to be virtualizable using a component. 
+
+**Dan Gohman**: We require that all APIs are defined in WIT. There is a possibility that APIs could be awkward to virtualize, but by requiring things be expressed in WIT then we know that everything can be virtualized. 
+
+**Dan Gohman**: I looked around and I found this unofficial rule that when something is posted we wait a week to vote on these things. We are not going to vote now, but we will vote on this process change next week. 2 weeks from now.
+
+**Pat Hickey**: I put this on the agenda.
+
+**Dan Gohman**: Should we document this 1 week wait before an agenda?
+
+**Conrad Watts**: Up to the subgroup on rules. CG follows one day beforehand for a poll. 
+
+**Deepti**: Just 24 hours before in CG.
+
+ **Pat Hickey**: OK then we can follow this in the future but not change for this vote.
+
+**Dan Gohman**: We intend for this process to not change.
```

Comment:
```suggestion
```

I think the context for this last remark isn't recorded here, so I think it's best to remove it.